extensions-core/s3-extensions/src/main/java/org/apache/druid/storage/s3/S3TransferConfig.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.storage.s3;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.validation.constraints.Min;
+
+/**
+ */
+public class S3TransferConfig
```

Comment:
we need to make sure we have sensible defaults if the user only enables `useTransferManager` without actually setting any of the other values.

Reply:
> we need to make sure we have sensible defaults if the user only enables `useTransferManager` without actually setting any of the other values.

I have verified that the default of 5MB works when only useTransferManager is set like the following
```
    # use AWS transfer manager to speed up uploads
    druid.storage.transfer.useTransferManager=true
    # druid.storage.transfer.minimumUploadPartSize=26214400
    # druid.storage.transfer.multipartUploadThreshold=26214400
```
Middle manager boots up with the following log message
```